lib/MetaCPAN/Web/Controller/Account.pm

```diff
@@ -52,21 +52,24 @@ sub profile : Local : Args(0) {
     return unless ( $req->method eq 'POST' );
 
     my $data = $author;
-    $data->{blog} = $req->param('blog.url')
+    $data->{blog}
```

Comment:
this is a weird tidyall issue (probably version mismatch), this change fails on Travis.